ui/src/shared/utils/analytics.ts

```diff
@@ -0,0 +1,25 @@
+export const fireOrgIdReady = function fireOrgIdReady(
+  organizationIds: string[]
+) {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push({
+    event: 'cloupAppOrgIdReady',
+    identity: {
+      organizationIds,
+    },
+  })
+}
+
+export const fireUserDataReady = function fireUserDataReady(
+  id: string,
+  email: string
+) {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push({
+    event: 'cloupAppUserDataReady',
+    identity: {
+      email,
+      id,
+    },
+  })
+}
```

Comment:
these two methods are somewhat special. the idea is that eventually we'll have a simple interface for firing arbitrary events that abstracts the implementation (currently it's google tag manager). right now, it's just these two methods, but i could see something like:

```ts
export const fireEvent = function fireEvent(eventName: string, contextData: any) {
  // implementation
}
...
componentDidMount() {
  fireEvent('data-explorer-loaded')
}
```

becoming more commonplace